CODE/bluetooth_logger.py

```
@@ -67,7 +68,7 @@ def run_powershell_command(command):
 
 
 # Unified parsing function for PowerShell output
-def parse_output(lines, regex, group_names):
+def parse_output(lines: list[LiteralString], regex: str, group_names: list[str]):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)